driver-sync/src/test/functional/com/mongodb/client/OcspTest.java

```diff
@@ -0,0 +1,58 @@
+package com.mongodb.client;
+
+import com.mongodb.MongoException;
+import com.mongodb.MongoNamespace;
+import com.mongodb.client.test.CollectionHelper;
+import org.bson.BsonDocument;
+import org.bson.BsonInt32;
+import org.bson.Document;
+import org.bson.codecs.DocumentCodec;
+import org.junit.Before;
+import org.junit.Test;
+
+import static com.mongodb.ClusterFixture.serverVersionAtLeast;
+import static com.mongodb.client.Fixture.getDefaultDatabaseName;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
+
+public class OcspTest {
+    private static final String COLLECTION_NAME = "test";
+    private final boolean shouldSucceed = Integer.parseInt(System.getenv("OCSP_TLS_SHOULD_SUCCEED")) == 1;
+    private CollectionHelper<Document> collectionHelper;
+
+    @Before
+    public void setUp() {
+        // Set the system properties necessary to enable OCSP.
+        System.setProperty("ocsp.enabled", "true");
+        System.setProperty("com.sun.net.ssl.checkRevocation", "true");
```

Comment:
I think we may also need `System.setProperty("jdk.tls.client.enableStatusRequestExtension", "false");`

Reply:
According to https://docs.oracle.com/en/java/javase/11/security/java-secure-socket-extension-jsse-reference-guide.html#GUID-F15D190D-85A1-4012-8FE3-060DBD90E579, the default value is true.

Reply:
We now have this set in a Gradle property in `run-ocsp-test.sh`.